go.mod

```diff
@@ -83,6 +83,7 @@ require (
 	go.opentelemetry.io/otel v1.30.0
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.30.0
 	go.opentelemetry.io/otel/sdk v1.30.0
+	go.uber.org/automaxprocs v1.6.0
```

Comment:
I think you should probably revert the changes in `go.mod` in order for there not to be any conflicts with `master`.

Reply:
this will lead to dependency issues, resulting in failure of CI checks.